features/gba-dev/overlijden-gba.feature

```diff
@@ -0,0 +1,127 @@
+# language: nl
+
+Functionaliteit: overlijden GBA: Altijd leveren van overlijdensdatum van een overleden persoon
+
+  Wanneer een consumer van de BRP API gegevens bevraagt van een overleden persoon
+  Dan levert de GBA api altijd de overlijdensdatum zodat de indicatieOverleden door de proxy op "true" gezet kan worden
+
+  Rule: indicatieOverleden heeft de waarde true als de betreffende persoon is overleden. Dit veld wordt altijd meegeleverd, ook als het niet met fields is gevraagd
```

Comment:
Ik denk dat deze rule verwarrend is. De GBA API kent geen indicatieOverleden veld.